src/adv.py

```diff
@@ -49,3 +66,74 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+class Quit(Exception):
+    pass
+
+# Accept inputs from the player
+
+
+def player_input(player):
+    player_input = input("🔮 Enter a command: ").lower()
+    command = player_input.split()[0]
+
+    if len(player_input.split()) == 2:
+        game_item = player_input.split()[1]
```

Comment:
Take another look at how you are initializing this variable `game_item`. 

When I try to use the `take`, `drop`, `look` command, I get the following error:
```UnboundLocalError: local variable 'game_item' referenced before assignment```